azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -49,7 +54,8 @@
   @Override
   public void uploadExecutableFlow(ExecutableFlow flow)
       throws ExecutorManagerException {
-    flows.put(flow.getExecutionId(), flow);
+    ExecutableFlow exFlow = ExecutableFlow.createExecutableFlowFromObject(flow.toObject());
```

Comment:
Which code path did you refer to?

How is it going to help readers understand why cloning here is needed?